AppCenter/AppCenterTests/MSAppCenterIngestionTests.m

```diff
@@ -693,4 +261,12 @@ - (void)testHideSecretInResponse {
   [mockLogger stopMocking];
 }
 
+#pragma mark - Test Helpers
+
+// TODO: Move this to base MSHttpIngestion test.
```

Comment:
Is it still valid?

Reply:
Looks like it is already there and this one is unused, so deleting.